### PR TITLE
Adding CosmosDB plugin UT + minor fix to DirectLine plugin UT

### PR DESCRIPTION
**Pull request by eladiw**

### CODE REVIEW

File: client/@types/botframework.d.ts [hunk 1]
```diff
@@ -34,7 +34,6 @@ interface BotFrameworkDataSource extends IDataSource {
   * A simple query on application insights data source
   */
 interface BotFrameworkQueryParams {
```
Comment (Ilanak) [inline client/@types/botframework.d.ts]: Is this interface in use?

Comment (eladiw) [inline client/@types/botframework.d.ts]: yes two lines above.

File: client/src/data-sources/plugins/CosmosDB/Query.ts [hunk 2]
```diff
@@ -68,11 +69,11 @@ export default class CosmosDBQuery extends DataSourcePlugin<IQueryParams> {
     };
 
     return (dispatch) => {
-      request(url, {
+      request(COSMOS_DB_QUERY_URL, {
         method: 'POST',
         json: true,
         body: body,
-      },      (error, json) => {
+      },      (error, json, b) => {
```
Comment (Ilanak) [inline client/src/data-sources/plugins/CosmosDB/Query.ts]: What is this 'b'?

Comment (eladiw) [inline client/src/data-sources/plugins/CosmosDB/Query.ts]: weird, should have been removed

Comment (eladiw) [inline client/src/data-sources/plugins/CosmosDB/Query.ts]: Fixed

**Pull request merged**